### PR TITLE
[DO NOT MERGE] Generated by websites-modules

### PR DESCRIPTION
**Pull request by guacbot**
Auto-generated by websites-modules github workflow.  
 Preview URL: https://docs-staging.datadoghq.com/websites-modules/generated/71

**What is this PR**
Moving header/footer data to website modules and changes to support loading it from one unified location

**Additional Notes**
- Remove main nav and footer nav from menu files and load from websites-modules instead
- Use shorter translations in get started to avoid design issues
- Update the footer html to load in data in the same time as corp
- delete old mobile nav menu

**What to test?**
1. Test that the header menu works/looks correct and that there are no broken links
2. Test that the footer menus works/looks correct and that there are no broken links
3. Test items 1. and 2. at mobile size
4. Test items 1. and 2. on the `/api/` section of the site as this loads in the api nav.
5. Test the above items in other languages `/ja/` and `/fr/`
6. Spot test random pages

### CODE REVIEW

File: config/_default/config.yaml [hunk 1]
```diff
@@ -37,10 +37,28 @@ module:
...
+     - source: layouts/partials/nav/nav-logo-download.html
+       target: layouts/partials/nav/nav-logo-download.html
+     - source: layouts/partials/nav/menu-mobile.html
+       target: layouts/partials/nav/menu-mobile.html
+     - source: layouts/partials/nav/mobile-documentation.html
+       target: layouts/partials/nav/mobile-documentation.html
+     - source: layouts/partials/search-bar.html
+       target: layouts/partials/search-bar.html
+     - source: layouts/partials/announcement_banner.html
+       target: layouts/partials/announcement_banner.html
```
Comment (nsollecito) [inline config/_default/config.yaml:51]: looks like this is repeated below

**Comment by nsollecito** [comment]
Looks like the Docs main sidenav is no longer scrollable causing items off screen to be inaccessible

**Comment by davidejones** [comment]
> Looks like the Docs main sidenav is no longer scrollable causing items off screen to be inaccessible

I believe this is fixed now

**Comment by davidejones** [comment]
just noticed a bug, we need to make sure if you are on a 2nd level nav item that it is highlighted and expanded showing the 3rd level too. Otherwise you can't reach certain pages. 
Screenshot 2022-02-02 at 10 34 25

**Comment by sarina-dd** [comment]
Only thing I noticed is a blog footer link: under blog, when you click JA and then try to click back to english, the link takes you back to JA. Otherwise, LGTM 👍

**Pull request merged**